File: app/src/main/java/io/stacknix/merlin/android/demo/models/Project.java
package io.stacknix.merlin.android.demo.models;

import io.stacknix.merlin.db.MerlinObject;
import io.stacknix.merlin.db.annotations.Ignore;
import io.stacknix.merlin.db.annotations.Model;
import io.stacknix.merlin.db.annotations.Order;
import io.stacknix.merlin.db.annotations.PrimaryKey;
import io.stacknix.merlin.db.annotations.SortKey;

@Model("vm.machine")
public class Project extends MerlinObject {
    public String name;
    public String ipv4;
}

File: app/src/main/java/io/stacknix/merlin/android/demo/models/Project.kt
package io.stacknix.merlin.android.demo.models

import io.stacknix.merlin.db.MerlinObject
import io.stacknix.merlin.db.annotations.Model

@Model("project")
open class Project(val long: Long, var name: String, var price: Float) : MerlinObject()
